core/src/main/java/apoc/trigger/TriggerHandler.java

```diff
@@ -114,9 +72,11 @@ private void updateCache() {
      * For most deployments this isn't an issue, since you can turn the config flag off, but in large fleet deployments
      * it's nice to have uniform config, and then the memory savings on databases that don't use triggers is good.
      */
-    private synchronized void reconcileKernelRegistration() {
+    public synchronized void reconcileKernelRegistration(boolean hasTriggers) {
+        lastUpdate = System.currentTimeMillis();
```

Comment:
Why do we still want a `lastUpdate`? That made sense only when there was a cache for the triggers

Also why did we get rid of the cache? Was that necessary?

Reply:
Yes true, indeed there is no more need currently. Removed